addons/website/static/src/interactions/header/base_header.js

```diff
@@ -0,0 +1,209 @@
+import { Interaction } from "@website/core/interaction";
+import { compensateScrollbar } from "@web/core/utils/scrolling";
+import { SIZES, utils as uiUtils } from "@web/core/ui/ui_service";
+
+export const extraMenuUpdateCallbacks = [];
+
+export default {
```

Comment:
don´t do default export, it rarely helps, is more often than not annoying, and definitely feel wrong here